unittests/autoconfig.py

```diff
@@ -56,6 +56,11 @@ class cxx_parsers_cfg(object):
         if 'msvc9' == gccxml.compiler:
             gccxml.define_symbols.append('_HAS_TR1=0')
 
+if cxx_parsers_cfg.gccxml.xml_generator:
```

Comment:
Not sure how to reproduce what you mean. Could you give me the configuration file you used ?